cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help

```diff
@@ -50,3 +50,5 @@ COMMANDS
         bindgen         Generate the Ballerina bindings for Java APIs
         shell           Run Ballerina interactive REPL [Experimental]
         version         Print the Ballerina version
+        profile         Run Ballerina profiler,
+                        generate flame graph [Experimental]
```

Comment:
Do we need "generate flame graph [Experimental]". 
Start Ballerina Runtime Profiler [Experimental]